opentargets.json

```
@@ -566,6 +566,98 @@
       ],
       "additionalProperties": false
     },
+    {
+      "properties": {
+        "datasourceId": {
+          "const": "gene_burden"
+        },
+        "ancestry": {
+          "$ref": "#/definitions/ancestry"
+        },
+        "ancestryId": {
+          "$ref": "#/definitions/ancestryId"
+        },
+        "beta": {
+          "$ref": "#/definitions/beta"
+        },
+        "betaConfidenceIntervalLower": {
+          "$ref": "#/definitions/betaConfidenceIntervalLower"
+        },
+        "betaConfidenceIntervalUpper": {
+          "$ref": "#/definitions/betaConfidenceIntervalUpper"
+        },
+        "cohortId": {
+          "$ref": "#/definitions/cohortId"
+        },
+        "datatypeId": {
+          "$ref": "#/definitions/datatypeId"
+        },
+        "diseaseFromSource": {
+          "$ref": "#/definitions/diseaseFromSource"
+        },
+        "diseaseFromSourceMappedId": {
+          "$ref": "#/definitions/diseaseFromSourceMappedId"
+        },
+        "literature": {
+          "$ref": "#/definitions/literature"
+        },
+        "oddsRatio": {
+          "$ref": "#/definitions/oddsRatio"
+        },
+        "oddsRatioConfidenceIntervalLower": {
+          "$ref": "#/definitions/oddsRatioConfidenceIntervalLower"
```

We don't distinguish OR and beta confidence intervals:
```suggestion
          "$ref": "#/definitions/confidenceIntervalLower"
```